mmv1/templates/terraform/post_create/compute_target_instance_security_policy.go.erb

```diff
@@ -0,0 +1,34 @@
+<% unless version == 'ga' -%>
+// security_policy isn't set by Create
```

Comment:
Just want to double-check that it can't be set on create? The API docs say that it should be included in the `insert` request body but also says it's output-only 😂  https://cloud.google.com/compute/docs/reference/rest/beta/targetInstances/insert#request-body

Reply:
No. The field is "ignored" in the request.